### ISSUE TITLE
Update grid on Vocabulary site

### ISSUE THREAD

**Issue by annatuma** [cc-archive/vocabulary-legacy#76]
**Description**
After we decide on a new Grid system to use, we need to update the grid on the Vocabulary site.

This ticket is blocked by the research phase:
https://github.com/creativecommons/vocabulary/issues/75

**Comment by kgodey** [comment]
Closing in favor of #75, which I updated with actually implementing the grid. We're no longer going to be researching, we'll just use Bulma.
